From: Peter Micahel Green <plugwash@raspbian.org>
Subject: Automatically generated patch (1:4.0.1-1+rpi1)

Last (up to) 3 git changes, FYI:

commit 67c335abb78deb53de96a26ff0f0f5ac8ffaabde
Author: Peter Michael Green <plugwash@raspbian.org>
Date:   Thu Jul 6 21:25:21 2017 +0000

    Remove old cmake-arm-version-hack.patch. dgit will regenerate it.

commit 019f07c5de27a04a420eb6889bbe96c9dca246d7
Merge: d19e83d0b d0f4f4a87
Author: Peter Michael Green <plugwash@raspbian.org>
Date:   Thu Jul 6 21:49:03 2017 +0100

    Merge raspbian changes from llvm-toolchain-3.9 packaging.

commit d19e83d0b14cf997e617ffbf125d744d1ac08100
Merge: b60f1e5d8 301c08848
Author: root <root@cuboxi4x4.Home>
Date:   Thu Jul 6 21:39:58 2017 +0100

    Psudeomerge to tie in llvm-toolchain-3.9 upstream history.
---

--- /dev/null
+++ llvm-toolchain-4.0-4.0.1/cmake/config.guess.hack
@@ -0,0 +1,2 @@
+#!/bin/bash -e
+$(dirname "$0")/config.guess | sed -r s/arm[a-zA-Z0-9]*/armv6/
--- llvm-toolchain-4.0-4.0.1.orig/cmake/modules/GetHostTriple.cmake
+++ llvm-toolchain-4.0-4.0.1/cmake/modules/GetHostTriple.cmake
@@ -15,7 +15,7 @@ function( get_host_triple var )
       set( value "i686-pc-mingw32" )
     endif()
   else( MSVC )
-    set(config_guess ${LLVM_MAIN_SRC_DIR}/cmake/config.guess)
+    set(config_guess ${LLVM_MAIN_SRC_DIR}/cmake/config.guess.hack)
     execute_process(COMMAND sh ${config_guess}
       RESULT_VARIABLE TT_RV
       OUTPUT_VARIABLE TT_OUT
--- llvm-toolchain-4.0-4.0.1.orig/compiler-rt/cmake/base-config-ix.cmake
+++ llvm-toolchain-4.0-4.0.1/compiler-rt/cmake/base-config-ix.cmake
@@ -170,8 +170,8 @@ macro(test_targets)
       if(WIN32)
         test_target_arch(arm "" "" "")
       else()
-        test_target_arch(arm "" "-march=armv7-a" "-mfloat-abi=soft")
-        test_target_arch(armhf "" "-march=armv7-a" "-mfloat-abi=hard")
+        test_target_arch(arm "" "" "-mfloat-abi=soft")
+        test_target_arch(armhf "" "" "-mfloat-abi=hard")
       endif()
     elseif("${COMPILER_RT_DEFAULT_TARGET_ARCH}" MATCHES "aarch32")
       test_target_arch(aarch32 "" "-march=armv8-a")
